specification/ai/Azure.AI.Unified/connections/models.tsp

```diff
@@ -0,0 +1,72 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@typespec/openapi";
+import "@typespec/versioning";
+import "../credentials/models.tsp";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Unified;
+
+@doc("Response from the listSecrets operation")
+model Connection {
+  @doc("A unique identifier for the connection")
+  @visibility("read")
+  id: string;
```

Comment:
Why do we have both "id" and "name"? which one uniquely defines the connection? is it the id? if so, can I have two connections with the same name? It would be best not to expose both if possible.